reliability-engineering/terraform/modules/concourse-base/variables.tf

```diff
@@ -14,6 +14,18 @@ variable "number_of_availability_zones" {
   default = 2
 }
 
+variable "openid_connect_provider_url" {
+  type = string
+}
+
+variable "openid_connect_provider_client_id" {
+    type = string
+}
+
+variable "openid_connect_provider_tls_cert_thumbprints" {
+  type = list(string)
+}
```

Comment:
Y'know if you gave these values empty defaults and used a `for_each` trick on the `aws_iam_openid_connect_provider.concourse_secrets_admin` we'd be able to merge this now and it would also mean the whole oidc secret access thing could remain "optional".

Reply:
Though this is out of date now that we do have the live instances up.

Reply:
I've pushed a change that should make these optional because I also realized not making this optional would give us an impossible release order of the two repos.